### ISSUE TITLE
Bootstrap/Ant style collisions

### ISSUE THREAD

**Issue by ranbena** [getredash/redash#3258]
Redash is in the midst of migrating from Bootstrap to Ant, and from [Angular to React](https://github.com/getredash/redash/issues/3071).
Loading both component libraries causes collisions, since both use some identical `less` variable names, resulting mostly in wrong colors being displayed.

Bootstrap cannot be fully swapped until React migration is complete (Ant doesn't offer Angular components) so a middle ground must be found.

Currently defining specific css overrides, which problematic in many ways. Need better solution.

**Comment by ranbena** [comment]
Seems importing `css` instead of `less` files in `ant.less` resolves this.
I think it's good enough for now.

**Comment by arikfr** [comment]
@ranbena the problem with importing the `css` version is that our overrides won't apply:

https://github.com/getredash/redash/blob/22f835d3cbf0088b86129b235a73d621e342d725/client/app/assets/less/redash/ant.less#L12-L42

No?

**Comment by gabrieldutra** [comment]
I agree with @arikfr on this, the issue still exists, although we can avoid it for some components that don't require variable override.

**Comment by gabrieldutra** [comment]
I tested migrating all `less` files related to #3209 and some differences are noticeable (e.g.: font for form labels and inputs). My first test was with buttons, so that wasn't annoying.

I think what we can try do to at least give it some time is somehow isolate Ant less processing from the rest. I did a few tests with `webpack.config.js` and it seemed possible. It seems to create a variable-isolated tree when I do sth like this:
```javascript
// webpack.config.js from line 29
  entry: {
    app: [
      "./client/app/index.js",
      "./client/app/assets/less/main.less",
      "./client/app/assets/less/redash/ant.less"
    ],
    server: ["./client/app/assets/less/server.less"]
  }
```
With that we can create a new tree without including bootstrap on those paths. WDYT? :smile:

**Comment by arikfr** [comment]
@gabrieldutra if this works, it seems like the best solution indeed.

**Comment by gabrieldutra** [comment]
@ranbena what do you think? I'm not very experienced with Less (or Redash styling files), but I can try to reorganize those files.

My concerns are all related to what we will see after this division. Till now we've been coding based on a mix of styles and once we properly divide it some page/component styles could break. A very good look on pages and components (mainly the ones which use Ant) will be important.

**Comment by arikfr** [comment]
Can we close this now?
